### DISCUSSION TITLE
Mailoney honeypot not implementedd DATA

### DISCUSSION THREAD

**Discussion by vasokolov** [telekom-security/tpotce#1627] [Q&A]
Hi there! I have some issue with mailoney honepot.
I am trying to test it with  simple smtp command.
I have a connection to mailoney via telnet and port 25. Everything well but i cannot send any data. What i am doing wrong?

> sokolov@astra:~$ telnet 172.31.3.185 25
Trying 172.31.3.185...
Connected to 172.31.3.185.
Escape character is '^]'.
220 mailrelay.local ESMTP Exim 4.69 #1 Fri, 26 Jul 2024 07:36:47 -0700
MAIL FROM: 123@123.com
250 Ok
RCPT TO: host1@host.ru
250 Ok
DATA
" not implementedd "DATA

**Comment by t3chn0m4g3** [comment]
I already opened an [upstream issue](https://github.com/phin3has/mailoney/issues/19), once fixed we will include it.

**Reply by vasokolov** [reply]
ok, thank u.
